public-ui/src/app/data-browser/cdr-version/cdr-version-info.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Output,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import { environment } from 'environments/environment';
+import { Configuration, DataBrowserApi } from 'publicGenerated/fetch';
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import * as ReactDOM from 'react-dom';
+import { BaseReactWrapper } from '../../data-browser/base-react/base-react.wrapper';
+const api = new DataBrowserApi(new Configuration({ basePath: environment.publicApiUrl }));
+
+const containerElementName = 'root';
+
+export const style = `
+.result-body-item {
+    padding: 6px 0;
+}
+.cdr-info {
+    font-size: 14px;
+}
+`;
+
+interface State {
+    numParticipants: any;
+    creationTime: any;
+    cdrName: any;
+}
+
+export class CdrVersionReactComponent extends React.Component<{}, State> {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+        numParticipants: '',
+        creationTime: '',
+        cdrName: ''
+    };
+  }
+
+  componentDidMount() {
+      this.getCdrInfo();
+  }
+
+  getCdrInfo() {
+    api.getCdrVersionUsed().then(
+          (result) => {
+            this.setState({
+            numParticipants: result.numParticipants,
+            creationTime: new Date(result.creationTime),
+            cdrName: result.name});
+          });
+  }
+
+  render() {
+    return !!this.state.creationTime && <React.Fragment>
+    <style>{style}</style>
+    <span className='result-body-item cdr-info'>
+      Data includes {Number(this.state.numParticipants).toLocaleString()}
```

Comment:
May need to add a `&nbsp;` here as there's no space when rendered:
<img width="640" alt="Screen Shot 2021-03-11 at 12 38 08 PM" src="https://user-images.githubusercontent.com/40036095/110837475-e4183980-8266-11eb-80a0-e33ce9c8c5aa.png">